### PR TITLE
Automate release process of ReSDK with GitHub actions

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -15,14 +15,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tox-env: [py36, py37, py38, docs, linters, packaging]
+        tox-env: [py36, py37, py38, py39, docs, linters, packaging]
```
Comment (dblenkus) [inline .github/workflows/ci.yaml:18]: Also, support 3.10, and remove support for 3.6 as it reached EOL.

And don't forget to add a change log for new/removed Python versions.

Comment (dblenkus) [reply .github/workflows/ci.yaml:18]: And don't forget to update versions here:
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/setup.py#L47
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/setup.py#L81-L83
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/tox.ini#L2
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/tox.ini#L29

Comment (JureZmrzlikar) [reply .github/workflows/ci.yaml]: Done.

Comment (gregorjerse) [reply .github/workflows/ci.yaml]: The problem is in genialis-base settings for resdk e2e tests which are used.
See https://github.com/genialis/genialis-base/pull/689

Comment (gregorjerse) [reply .github/workflows/ci.yaml]: Please also change `README.rst` and `docs/contributing.rst` to reflect the removal of support for `Python 3.6`.

**Review by dblenkus — APPROVED** [review]

**Pull request merged**